extensions/vertx/runtime/src/main/java/io/quarkus/vertx/ConsumeEvent.java

```diff
@@ -7,17 +7,23 @@
 import java.lang.annotation.Target;
 
 import io.vertx.core.eventbus.MessageCodec;
+import io.vertx.mutiny.core.eventbus.Message;
 
 /**
  * Marks a business method to be automatically registered as a Vertx message consumer.
  * <p>
- * The method must accept exactly one parameter. If it accepts {@link io.vertx.core.eventbus.Message} or
- * {@link io.vertx.mutiny.core.eventbus.Message} then the return type must be void. For any other type the
- * {@link io.vertx.core.eventbus.Message#body()} is passed as the parameter value and the method
- * may return an object that is passed to {@link io.vertx.core.eventbus.Message#reply(Object)}, either directly or via
- * {@link java.util.concurrent.CompletionStage#thenAccept(java.util.function.Consumer)} in case of the method returns a
- * completion stage.
- * 
+ * The method can accept the following parameter types:
+ * <ul>
+ * <li>{@link io.vertx.core.eventbus.Message} or {@link io.vertx.mutiny.core.eventbus.Message}</li>
+ * <li>{@code T} ({@link Message#body() message body})</li>
+ * <li>{@link Message#headers() MultiMap (message headers)}, {@code T}</li>
```

Comment:
Hm, I think that we should clarify the required number of params and ordering. E.g. currently, we require the headers param to be the first.